### ISSUE TITLE
CopyButton component alt text

### ISSUE THREAD

**Issue by mekaufman** [IBM/carbon-components-react#1973]
Related issue: https://github.ibm.com/Bluemix/core-dev/issues/7928

## Title line template: CopyButton component in carbon-components-react alt text

We are using the CopyButton component in the resource list details pane. When a user hovers on the icon, the alt text is not displayed. However, when you inspect fully, you do see there is alt text on the svg. I'm not sure if it's just in the wrong place to be able to be displayed via hover. Also, I haven't tested using JAWS (screen reader), but if it's not read out, then we have an a11y violation. I'll try to test that and post a comment in the issue.

![image](https://user-images.githubusercontent.com/13648625/53834033-054b3e80-3f58-11e9-900a-8d4e2ce9f78b.png)

Can we get an update to the component to have alt text show on hover?

**Comment by dakahn** [comment]
Possibly related to #1964
